.github/workflows/autofix-reviews.yml

```
@@ -0,0 +1,47 @@
+            4. Run `npm run lint` and `npm run build` to verify fixes don't break anything
+            5. Commit all fixes in a single commit with message "fix: address CodeRabbit review"
+            6. Push to the PR branch
+
+            Important:
+            - Only fix issues flagged by CodeRabbit — don't make unrelated changes
+            - If a suggestion is wrong or doesn't apply, skip it
+            - Always verify with lint and build before committing
+            - Use `escapeHtml()` for any user input in HTML templates (security requirement)
+            - Use explicit null checks (`!= null`) not truthy checks for Decimal fields
+            - Always update Zod schemas in src/lib/validators.ts when adding new fields
+          direct_prompt: |
```

actionlint (1.7.11)

[error] input "direct_prompt" is not defined in action "anthropics/claude-code-action@v1". available inputs are "additional_permissions", "allowed_bots", "allowed_non_write_users", "anthropic_api_key", "assignee_trigger", "base_branch", "bot_id", "bot_name", "branch_name_template", "branch_prefix", "claude_args", "claude_code_oauth_token", "exclude_comments_by_actor", "github_token", "include_comments_by_actor", "include_fix_links", "label_trigger", "path_to_bun_executable", "path_to_claude_code_executable", "plugin_marketplaces", "plugins", "prompt", "settings", "show_full_output", "ssh_signing_key", "track_progress", "trigger_phrase", "use_bedrock", "use_commit_signing", "use_foundry", "use_sticky_comment", "use_vertex"

(action)